src/workspacer.Native/Native/WindowsWindow.cs

```diff
@@ -163,7 +163,7 @@ public void Focus()
             {
                 Logger.Debug("[{0}] :: Focus", this);
                 Win32Helper.ForceForegroundWindow(_handle);
-                WindowFocused?.Invoke();
+                Focused?.Invoke();
```

Comment:
These changes here looks mostly like change in form, not in function. And they also look like they might break compatibility. Do we really need these changes?